charts/k8s-service/values.yaml

```diff
@@ -372,6 +372,26 @@ additionalContainerEnv: {}
 #         envVarName: CONFIG_FOO
 configMaps: {}
 
+# persistentVolumes is a list of maps that specifies PeristantVolumes that should be mounted on the pod.
```

Comment:
i guess it's not really a list of maps, more just like an object whose keys are minimal PersistentVolume specs

not sure if i have a strong inclination on if `list of objects` or a nested `object` definition is more intuitive; just make it consistent

Reply:
Ironically, this **is** (and kind of needs to be) a list of maps now, as a result of modifying our own stuff to support multilple persistant volumes. I found it was easier to use if it's a list of objects